app/javascript/components/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Greet from './Greet';
+
+function App() {
+  return (
+    <Routes>
+      <Route path="/" element={<Greet />} />
+    </Routes>
```

Comment:
there is an `App` component with routes 🙌 Great work!

Reply:
Cheers 👍🏽